### ISSUE TITLE
[BUG]: Formsy-react is using state._value but formsy-material-ui using state.value

### ISSUE THREAD

**Issue by Stupidism** [mbrookes/formsy-material-ui#133]
I found the bug in your source code because Chrome auto-filled but Formsy-react not recieving onChange event. I think it's a bug widely ranged in this repo that guys like me who don't know well about this repo can't fix it totally.
## Cause:

Chrome autofill triggers [change event](http://stackoverflow.com/questions/11708092/detecting-browser-autofill)
Formsy-react is using [this.state._value](https://github.com/christianalfoni/formsy-react/blob/master/src/Mixin.js#L120)
But formsy-material-ui uses [setState({value})](https://github.com/mbrookes/formsy-material-ui/blob/master/src/FormsyText.jsx#L65) when recieving onChange event
## Quick fix for FormsyText:

```
  handleChange: function handleChange(event) {
    ...
    this.setState({
      value: event.currentTarget.value,
    });
    // change to
    this.setValue(event.currentTarget.value);
    ...
  },
  ...
  render() {
    ...
        value={this.state.value}
        // change to
        value={this.state._value}
    ...
  }
```

**Comment by stupidisum** [comment]
I think there are some related issues:
- #132

**Comment by achtan** [comment]
can u pls fix this

**Comment by Stupidism** [comment]
@achtan It seems this repo is not maintained now. I'm considering use redux-form instead.

**Comment by rojobuffalo** [comment]
This looked like a pretty well explained bug that should be fixed. I'm going to reopen. Also, I've seen no evidence that formsy-react is no longer being maintained. Can anyone else lend any credibility to that claim?

**Comment by Stupidism** [comment]
Sorry, I wanted to say `formsy-material-ui` is not maintained, at least, community is not active. As I can see, #97 was issued on May, and it's not fixed yet, which is actually an important bug.

**Comment by rojobuffalo** [comment]
#125 is closed. Did #109 not get it right?

**Comment by rojobuffalo** [comment]
Hey @Stupidism, in your quick fix example, it doesn't look like you `setState({_value:...`. Was that a typo? Or where would you set `_value`?

**Comment by Stupidism** [comment]
`this.setValue` is the same as `this.setState({ _value })`. It's something from `formsy-react`

**Comment by Stupidism** [comment]
Actually, I don't see necessity to use `this.state.value` from my glimpse. I think maybe it's a legacy when `formsy-react` change from `value` to `_value`, which is a good change, but `formsy-material-ui` didn't follow up. Major bugs happend and someone found that add `this.setValue() with this.setState({value})` together wound fix it. As a result, it's not fixed in somewhere, like here. And that's why I think this repo is not maintained well now.

**Comment by Stupidism** [comment]
Sorry for my arbitrary words before. that's what I was thinking when creating this issue. 

I took a look at the current code and latest commits. You are doing a great job, saving a dying repo. @rojobuffalo 

Now I find that this issue is somehow solved already. The original title was `Chrome auto-filled but Formsy-react not recieving onChange event. `.

This issue was one of two things that bothered me very much. The other was that one day I found `required` error not showing anymore. I think I can close this issue and follow #179 instead
